platform/components/argilla/create-argilla-dataset.tsx

```diff
@@ -135,6 +143,37 @@ const CreateDataset = () => {
                     <DatePickerWithRange className="mr-2" />
                     <FilterComponent variant="tasks" />
                 </div>
+                {/* <div className="items-top flex space-x-2 mt-4">
+                    <Checkbox
+                        id="terms1"
+                        checked={useSmartSampling}
+                        onCheckedChange={handleSmartSamplingChange}
+                    />
+                    <div className="grid gap-1.5 leading-none">
+                        <label
+                            htmlFor="terms1"
+                            className="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+                        >
+                            Force balanced dataset
+                        </label>
+                    </div>
+                </div> */}
+                <div className="mt-4">
```

Comment:
Should we add a nice message here to say how many are selected and if there are too many we disable the button and ask to filter instead of truncating ?